common-prod/common.tfvars

```diff
@@ -24,6 +24,21 @@ aws_account_ids = {
   cloud-platform             = "754256621582"
 }
 
+## cr = community rehabilitation
+cr_account_ids = {
+  hmpps-cr-jira-non-production        = "529698415668"
```

Comment:
Do we need both nonprod and prod in this list or just prod envs?

Reply:
in my use case yes. As I need to create a policy to enable access from one to the other.
So would reference the key for each account required.
Same as was_accounts above.
Thanks for checking.